public/style.css

```diff
@@ -49,7 +49,7 @@
     margin-left: -25px !important;
 }
 
```

Comment:
I'm getting a strange 'jump' in the line numbers on the side when I open up a report with a long query - is this something on the codemirror side or conflicting style from NeoDash?
https://user-images.githubusercontent.com/1754968/210563106-e77dfb42-3cef-47fe-82c3-7d193059d85a.mov